packages/core/networks/137.json

```diff
@@ -21,38 +21,14 @@
   },
   {
     "contractName": "Store",
-    "address": "0x2d4710F04Da90184255782d3715224A6C776955D"
+    "address": "0xE58480CA74f1A819faFd777BEDED4E2D5629943d"
   },
   {
     "contractName": "OptimisticOracle",
-    "address": "0x8A481F28C23c72CB01452ca570ba165DB5F55D24"
+    "address": "0xAaF3F29521877d7a28C21E379Bf7A5401883A6B2"
   },
   {
-    "contractName": "TokenFactory",
```

Comment:
Removed these contracts for now because we're not officially supporting them yet.

Reply:
When are we planning on supporting them?

Reply:
Whenever we test the bots out some more, then we should redeploy because the `FundingRateApplier` code changed a bit with stamping ancillary data logic. Should be easy to support